src/makeInfinityAdder.js

```diff
@@ -4,7 +4,26 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0; // Initialize the sum to 0
+
+  function adder(num) {
```

Comment:
The `adder` function should handle the case where `num` is not a number. Consider adding a check to ensure `num` is a valid number before adding it to `sum`.